### PR TITLE
meson 1.1.1

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 1.1.1

https://github.com/mesonbuild/meson/tree/1.1.1

### CODE REVIEW

**Review by varlackc — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,7 +25,6 @@ requirements:
   build:
     - patch     # [not win]
     - m2-patch  # [win]
-
   host:
     - python
     - setuptools >=42
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:30]: Could you use `ninja-base` instead of `ninja`?

**Review by skupr-anaconda — APPROVED** [review]
LGTM but ninja

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml:30]: There is no impact here as meson has python as run dependency anyway.

**Pull request merged**